_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,235 @@
+`-s`, `--from` | The email address of the sender. | For example, `user@amazon.com`. | OPENSEARCH_FROM
+`-r`, `--to` | The email address of the recipient. | For example, `user@amazon.com`. | OPENSEARCH_TO
+`--smtphost` | The hostname of the SMTP server. | For example, `SMTP_HOST`. | OPENSEARCH_SMTP_HOST
+`--smtpport` | The port for connection. | For example, `SMTP_PORT`. | OPENSEARCH_SMTP_PORT
+`--smtpsecure` | Use TLS when connecting to server. | For example, `SMTP_SECURE`. | OPENSEARCH_SMTP_SECURE
+`--smtpusername` | The SMTP username.| For example, `SMTP_USERNAME`. | OPENSEARCH_SMTP_USERNAME
+`--smtppassword` | The SMTP password.| For example, `SMTP_PASSWORD`. | OPENSEARCH_SMTP_PASSWORD
+`--subject` | Email Subject text encased in quotes. | Can be any string. The default is "This is an email containing your dashboard report". | OPENSEARCH_EMAIL_SUBJECT
+`-h`, `--help` | Display list of optional arguments from the command line. | N/A
+
+
+
+## Use environment variables with Reporting CLI
+
+Instead of explicitly providing values in the command line, you can save them as environment variables. The Reporting CLI reads environment variables from the current directory inside the project.
+
+To set the environment variables on Linux, use the following command. 
+
+```
+export NAME=VALUE
+```
+
+Each line should have the format `NAME=VALUE`.
+Each line that starts with a hashtag (#) is considered as a comment.
+Quotation marks (") don't get any special handling.
+
+Values from the command line argument have higher priority than the environment file. For example, if you add filename as *test* in *.env* file and also add `--filename report` command option, the generated report's name will be *report*.
+{: .note }
+
+### Example: Request a PNG report with environment variables set
+
+The following command requests a dashboard report with basic authentication in png format.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --format png --auth basic --credentials admin:admin
+```
+
+Upon success, the report will download to the current directory.
+
+### Use Amazon SES to request an email with a report attachment
+
+To use Amazon SES as the email transport mechanism, the following prerequisites apply:
+
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily. The sender's email address must be verified on [Amazon SES](https://aws.amazon.com/ses/).
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.
+ [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) is required to interact with Amazon SES.
+ [Configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) basic settings used by AWS CLI.
```

Comment:
```suggestion
 [Configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) basic settings used by the AWS CLI.
```

Reply:
yes. see updated direct link to AWS CLI user guide.